astropy/io/fits/tests/test_core.py

```diff
@@ -675,8 +675,12 @@ def test_open_from_remote_url(self):
 
             except urllib.error.HTTPError:
                 continue
+            except urllib.error.URLError:
```

Comment:
why not combine the two exceptions? `except (urllib.error.HTTPError, urllib.error.URLError)`

p.s. Great to get rid of these annoying failures!

Reply:
@mhvk - done!